nifi-docs/src/main/asciidoc/toolkit-guide.adoc

```diff
@@ -106,6 +106,12 @@ The following are available commands:
  registry import-flow-version
  registry sync-flow-versions
  registry transfer-flow-version
+ registry create-user
+ registry update-user
+ registry create-user-group
+ registry update-user-group
+ registry create-policy
```

Comment:
Lets update this to be 'get-policy' since we got rid of create and folded into update.